students/Net_Michael/lesson3/seq_list.py

```diff
@@ -0,0 +1,28 @@
+#!/usr/bin/env python
+# coding: utf-8
+
+def exchange_first_last(seq):
+    a_new_sequence = seq[-1:] + seq[1:-1] + seq[:1]
+    return(a_new_sequence)
+
+def mid_last_first(seq):
+    if type(seq) is str:
+        seq = seq.split()
+        ll = len(seq)
+        m = round(ll/2) - 1
+        a_new_sequence =  seq[m:] + seq[:m]
+        a_new_sequence = " ".join(a_new_sequence)
+    elif type(seq) is tuple:
+        ll = len(seq)
+        m = round(ll/2) - 1
+        a_new_sequence =  seq[m:] + seq[:m]
+    return (a_new_sequence)
```

Comment:
this is incorrect - this should be using slicing only and not checking for any specific type.